components/accelerator/nvidia/ecc/component.go

```diff
@@ -67,14 +67,14 @@ func (c *component) Start() error {
 	return nil
 }
 
-func (c *component) States(ctx context.Context) ([]apiv1.State, error) {
+func (c *component) HealthStates(ctx context.Context) (apiv1.HealthStates, error) {
```

Comment:
Some components return apiv1.HealthStates, others return []apiv1.HealthState. Should we keep the same style?